### PR TITLE
BZ#2051835 - fix typo in DHCP note

### PR DESCRIPTION
**Pull request by aireilly**
Fixes typo introduced with https://github.com/openshift/openshift-docs/pull/41781 to address [BZ#2051835](https://bugzilla.redhat.com/show_bug.cgi?id=2051835)

Preview: https://deploy-preview-41938--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html#network-requirements-reserving-ip-addresses_ipi-install-prerequisites

Merge to main, enterprise-4.8, enterprise-4.9, enterprise-4.10.

### CODE REVIEW

File: modules/ipi-install-network-requirements.adoc [hunk 1]
```diff
@@ -110,7 +110,7 @@ Some administrators prefer to use static IP addresses so that each node's IP add
 [IMPORTANT]
 .Ensuring that your DHCP server can provide infinite leases
 ====
-Your DHCP server must provide a DHCP expiration time of 4294967295 seconds to properly set an infinite lease as specified by link:https://datatracker.ietf.org/doc/html/rfc2131[rfc2131]. If a lesser value is returned for the DHCP infinite lease time, the node reports an error and a static IP is not set for the node. In RHEL 8, `dhcpd` does not provides infinite leases. If you want to use the provisioner node to serve dynamic IP addresses with infinite lease times, use `dnsmasq` rather than `dhcpd`.
+Your DHCP server must provide a DHCP expiration time of 4294967295 seconds to properly set an infinite lease as specified by link:https://datatracker.ietf.org/doc/html/rfc2131[rfc2131]. If a lesser value is returned for the DHCP infinite lease time, the node reports an error and a static IP is not set for the node. In RHEL 8, `dhcpd` does not provide infinite leases. If you want to use the provisioner node to serve dynamic IP addresses with infinite lease times, use `dnsmasq` rather than `dhcpd`.
```
Comment (reihl) [inline modules/ipi-install-network-requirements.adoc:113]: I don't think infinite lease is equal to static IP, the method under the interface is stay auto and there is a chances that the node will request from the dhcp a new address after a while due to problems on the node itself (and indeed, the dhcp will answer with the same address as before)

Comment (aireilly) [reply modules/ipi-install-network-requirements.adoc:113]: Thanks @reihl! @dmoessne Can you comment on this? Do we need to change this note?

Comment (dmoessne) [reply modules/ipi-install-network-requirements.adoc:113]: Right, an infinite lease is in dhcp context not a static IP on the other hand, this is to my understanding also not said here. If the dhcp server sends out infinite leases during cluster install time the RHCOS installer will observe this and use it as a trigger for writing down that ip statically on the node. Hence after the install is complete, the dhcpd can removed as the nodes has configured the IP it got form the dhcp server with an infinite lease as static. Also see https://bugzilla.redhat.com/show_bug.cgi?id=2048887#c7 
but if it helps we possibly could rephrase like " If a lesser value is returned for the DHCP infinite lease time, the node reports an error and will not set the dhcp  provided IP as static IP for the node"

Comment (reihl) [reply modules/ipi-install-network-requirements.adoc:113]: In the first sentence, switch "static" with "permanent" and this is perfect

Comment (reihl) [reply modules/ipi-install-network-requirements.adoc]: :+1:

**Comment by aireilly** [comment]
> @aireilly LGTM! Why shouldn't the change be applied to 4.10? Looks like the same typo was introduced in #41896. Thanks!

merge to 4.10 too please for good measure Ashley :)

**Comment by ahardin-rh** [comment]
4.8 CP: https://github.com/openshift/openshift-docs/pull/42008

**Pull request merged**